src/bootstrap.js

```diff
@@ -46,8 +46,12 @@ function reset() {
       } else {
         const benchmark = benchmarks[index];
         text += `<td class="benchmark-name">`;
-        text += `<a href="https://github.com/v8/web-tooling-benchmark/blob/master/docs/in-depth.md#${benchmark.name}" target="_blank">${benchmark.name}</a></td>`;
-        text += `<td class="result" id="results-cell-${benchmark.name}">&mdash;</td>`;
```

Comment:
These changes seem unrelated, can you undo them?

Reply:
It might've been `prettier` making these changes.